src/components/editor/Bindings/Backfill/index.tsx

```diff
@@ -36,7 +36,7 @@ function Backfill({ description, bindingIndex = -1 }: BackfillProps) {
     // Binding Store
     const currentCollection = useBinding_currentCollection();
     const currentBindingUUID = useBinding_currentBindingUUID();
-    const collectionsCount = useBinding_collections_count();
+    const collectionsCount = useBinding_enabledCollections_count();
```

Comment:
Want to disable the "backfill all" button only when there are no enabled bindings